pages/js/frameworks.md

```diff
@@ -5,6 +5,14 @@ layout: docs
 sidenav: js
 ---
 # Frameworks
+
+**We recommend using [React]() for projects when possible.** As a team, we enjoy using React because it has a strong component model, enables easy server side rendering, and faciliates debugging by making the UI elements a function of the application state. React is most akin to the "view layer" in an MVC (Model-View-Controller) architecture and is frequently used with some other libraries. In our projects, we tend to use:
+
+* [Redux](https://www.npmjs.com/package/redux) for managing state
+* [Webpack](https://www.npmjs.com/package/webpack) and [Babel](https://babeljs.io/) for compiling our Javascript
```

Comment:
Does [react-router](https://reacttraining.com/react-router/) belong on this list, in a similarly "optional" bullet as Redux?  Do we have a different routing library we like to use?  Do we tend to roll our own (please no)?

Reply:
Also curious if we want to recommend something like [create-react-app](https://github.com/facebook/create-react-app) to help bootstrap a new React project.  Goodness knows setting up Webpack by hand is no fun.  😄